tests/backup/backup_locked_bucket_test.go

```diff
@@ -178,7 +178,7 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 
 		log.InfoD("Deleting backup location and cloud setting")
 		for backupLocationUID, backupLocationName := range BackupLocationMap {
-			err := DeleteBackupLocation(backupLocationName, backupLocationUID, orgID)
+			err := DeleteBackupLocation(backupLocationName, backupLocationUID, orgID, false)
```

Comment:
Here should it be err := DeleteBackupLocation(backupLocationName, backupLocationUID, orgID, true)?

Reply:
@snigdha-px It's a locked bucket you can't remove backups before 2 days.

Reply:
ok